pandas/core/groupby/generic.py

```diff
@@ -1786,7 +1890,7 @@ def _normalize_keyword_aggregation(kwargs):
     """
     Normalize user-provided "named aggregation" kwargs.
 
-    Transforms from the new ``Dict[str, NamedAgg]`` style kwargs
+    Transforms from the new ``Mapping[str, NamedAgg]`` style kwargs
```

Comment:
I've been seeing typing-based names creep into docstrings, but it isn't obvious that these are more user-friendly.  Is this more-correct or otherwise a clear improvement?